GitUI/CommandsDialogs/RevisionFileTreeControl.cs

```diff
@@ -212,6 +215,8 @@ public void LoadRevision(GitRevision? revision)
 
                 if (tvGitTree.SelectedNode is null)
                 {
+                    BlameControl.Visible = false;
+                    FileText.Visible = true;
                     FileText.Clear();
```

Comment:
What do you think about having a separate method to control these `BlameControl `and `FileText `Visible states? I can see in a couple of instances you setting these two to false and true respectively.

Reply:
Similar in FileTree and RevDiff, just one of the controls are active at a time.
In this case there are only two controls and only true/false switch so I prefer to not hide this in a method.
For RevDiff, I separated `BlameSelectedFileDiffAsync()` to a separate method that was kept close to the existing `ShowSelectedFileDiffAsync()` to show the similarities.
(Good suggestion still...)